docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Serial Vault User Guide
+table_of_contents: true
+---
+
+# Serial Vault user guide
+
+The Serial Vault is a Canonical-hosted web service that is required to use a
+[Secure Brand Store][secure-brand-store]. The Serial Vault provides :
+
+ - Signed [serial assertions][serial-assertions]  to authenticated systems of
+   a given brand/model, which then use it to access a secure Brand Store.
+ - Signed [system-user assertions][system-user-assertions], which allow
+   creating a system user account from a USB drive on an unmanaged Core system.
```

Comment:
What exactly is an unmanaged Core system? This might confuse a new user.

Reply:
unfortunately explaining what an unmanaged core system is requires pulling in too much, in my opinion. Given that this point is not related to the main purpose, I settled for just mentioning it here (and not explaining it). For the record, it is a Core system that has had no system user added by assertion, whether by preseeding, console-conf or auto import from USB.

Reply:
Why not just mention the short sentence that you just did (i.e. A Core system that has had no system user added by assertion.)?